<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>918</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SVI Model</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCustomPlot" name="plot" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>461</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>340</y>
      <width>361</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-50</number>
    </property>
    <property name="maximum">
     <number>50</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="sliderPosition">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>380</y>
      <width>361</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-50</number>
    </property>
    <property name="maximum">
     <number>50</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="sliderPosition">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>420</y>
      <width>361</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-50</number>
    </property>
    <property name="maximum">
     <number>50</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="sliderPosition">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>460</y>
      <width>361</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-50</number>
    </property>
    <property name="maximum">
     <number>50</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="sliderPosition">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>500</y>
      <width>361</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-50</number>
    </property>
    <property name="maximum">
     <number>50</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="sliderPosition">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLCDNumber" name="param1">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>70</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Light">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Light">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Light">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="digitCount">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="param2">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>110</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Light">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Light">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Light">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="digitCount">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="param3">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>150</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Light">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Light">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Light">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="digitCount">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="param4">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>190</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Light">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Light">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Light">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="digitCount">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="param5">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>230</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Light">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Light">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Light">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="digitCount">
     <number>5</number>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>30</y>
      <width>171</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="plot2" native="true">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>300</y>
      <width>461</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>90</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Calibrate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sliders">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>190</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Sliders</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_speedChange">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>530</y>
      <width>48</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="value">
     <number>3</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_arbitrage">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>240</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove Arbitrage</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>70</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>a :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>110</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>b :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>150</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>m :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>190</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ro :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>230</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>sig :</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>918</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menumode">
    <property name="title">
     <string>Import</string>
    </property>
    <addaction name="actionAddFile"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>Plot</string>
    </property>
    <addaction name="actionset_range"/>
    <addaction name="actionset_initial_curve"/>
   </widget>
   <addaction name="menumode"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAddFile">
   <property name="text">
    <string>data set</string>
   </property>
  </action>
  <action name="actionarbitrage">
   <property name="text">
    <string>arbitrage</string>
   </property>
  </action>
  <action name="actionset_range">
   <property name="text">
    <string>set range</string>
   </property>
  </action>
  <action name="actionset_initial_curve">
   <property name="text">
    <string>set initial curve</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
